packages/contracts-bedrock/contracts/test/Encoding.t.sol

```diff
@@ -15,15 +16,17 @@ contract Encoding_Test is CommonTest {
 
     function test_encodeDepositTransaction() external {
         bytes memory raw = Encoding.encodeDepositTransaction(
```

Comment:
We need to delete this test because the encoding changed and now we have coverage via differential testing

Reply:
This test will break once this is rebased, i'm ok with re-adding it in the future